### ISSUE TITLE
Library will not finalize an order due to ACMEv2 change (which is in production)

### ISSUE THREAD

**Issue by ghost** [yourivw/LEClient#39]
As of Thursday, July 5th a new order status has been put in production, this means that orders will get a ready status. It appears that orders which have this status fail to finish with the current version of this library.

https://community.letsencrypt.org/t/acmev2-order-ready-status/62866

**Comment by ghost** [comment]
It appears that the only fix needed is to change line 512 of LEOrder.php (at least this fixed the issue on my end while testing/debugging.

```
		if($this->status == 'pending')
```

to

```
		if($this->status == 'pending' || $this->status == 'ready') 
```

**Comment by yourivw** [comment]
Should have been resolved by merging pull requests, now released in [v1.1.3](https://github.com/yourivw/LEClient/releases/tag/1.1.3).
